src/main/tests/BaseTest.java

```diff
@@ -0,0 +1,70 @@
+package main.tests;
+
+import main.pages.LoginPageObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+
+import java.util.concurrent.TimeUnit;
+
+public class BaseTest {
+   WebDriver driver;
+
+   @BeforeClass
+   @Parameters({"browser", "baseUrl"})
+   void setUp(@Optional("chrome") String browser, String baseUrl){
+       if (browser.equalsIgnoreCase("chrome")){
```

Comment:
```suggestion
       if (Constant.Chrome.equalsIgnoreCase( browser )
```
Use Constants file to maintain constants.
Create a separate file for URL constants.

Reply:
Will be listing all the constants such as url, username, email, password and other variable that could be needed for the test in config or separate file